src/dragonball/src/metric.rs

```diff
@@ -55,4 +58,6 @@ pub struct DragonballMetrics {
     pub seccomp: SeccompMetrics,
     /// Metrics related to signals.
     pub signals: SignalMetrics,
+    /// Metrics related to balloon device.
+    pub balloon: HashMap<String, Arc<BalloonDeviceMetrics>>,
```

Comment:
There will be at least eleven `RwLock ` for the metric since we have various device types, which may increase overhead. And the write operation is only used when creating or removing devices, it has low competitive overhead. So I think one big RwLock may be able to handle it.